### PR TITLE
[MU4] Extend slur to next system fix

### PR DESCRIPTION
**Pull request by Eism**
Resolves: #10741

### CODE REVIEW

**Review by RomanPudashkin — APPROVED** [review]

**Comment by DmitryArefiev** [comment]
Tested #10741 LinuxMint 20.2 - FIXED      

Please merge into master

File: src/framework/accessibility/tests/accessibilitycontroller_tests.cpp [hunk 1]
```diff
@@ -175,3 +188,36 @@ TEST_F(AccessibilityControllerTests, SendEventOnFocusChanged)
...
+
+    //! [GIVEN] Two items
+    AccessibleItem* item1 = make_item();
+    AccessibleItem* item2 = make_item();
+
+    //! [GIVEN] Register items
+    m_controller->reg(item1);
+    m_controller->reg(item2);
+
+    notExpectDispatchEventOnFocus();
```
[the remark this answers is not in the document]
Comment (Jojo-Schmitz) [reply src/framework/accessibility/tests/accessibilitycontroller_tests.cpp:208]: See [db5c909](https://github.com/musescore/MuseScore/pull/10711/commits/db5c9098a2a44014759e006a84d9417b877ec603)

[the remark this answers is not in the document]
Comment (Jojo-Schmitz) [reply src/framework/accessibility/tests/accessibilitycontroller_tests.cpp:208]: Or better df9149f

[the remark this answers is not in the document]
Comment (Eism) [reply src/framework/accessibility/tests/accessibilitycontroller_tests.cpp:208]: yes, it is better
thanks!

**Pull request merged**